src/Symfony/Component/HttpFoundation/Request.php

```diff
@@ -312,7 +312,7 @@ public static function create($uri, $method = 'GET', $parameters = array(), $coo
             'SERVER_NAME' => 'localhost',
             'SERVER_PORT' => 80,
             'HTTP_HOST' => 'localhost',
-            'HTTP_USER_AGENT' => 'Symfony/3.X',
+            'HTTP_USER_AGENT' => 'Symfony/4.X',
```

Comment:
...and the usual question when we do changes like this: can we remove the version number? would it make sense to do that? is it valid as a User Agent string?

Reply:
:+1: for removing the version